### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by veekeys**

### CODE REVIEW

**Review by bentleyvk — APPROVED** [review]

**Review by mayank99 — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: "Bug report \U0001F41E"
```
Comment (mayank99) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]: Wait, can you use the same emoji here as React?

Comment (veekeys) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: I think I did.. idk, why it doesnt show here...

Comment (veekeys) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: image

It shows, if you go to md..

Comment (veekeys) [reply .github/ISSUE_TEMPLATE/bug_report.md]: changed

**Review by mayank99 — APPROVED** [review]

**Pull request merged**